index.html

```diff
@@ -370,20 +382,24 @@ <h3>Use Cases and Requirements</h3>
         </li>
         <li>
 <a>Holders</a> are be able to freely choose and change the agents they employ
-to help them manage and share their <a>verifiable credentials</a>.
+to help them manage their <a>verifiable credentials</a> and generate and share <a>verifiable presentations</a>.
         </li>
         <li>
-<a>Holders</a> that share <a>verifiable credentials</a> are not required to
+<a>Holders</a> that share <a>verifiable presentations</a> never need to
```

Comment:
I think `are not required` is better language here. There are circumstances where it may even be desirable for holders to do so. I realize that "never need" may cover that but it's more of a jump, IMO.

Reply:
The intention was to strongly discourage the sharing of credentials but still keep the possibility.

Reply:
I like Lovesh's language better, because I think the spec should be discouraging direct sharing. Buf if that's not the group's consensus, we can leave the weaker form. @dlongley , is your feedback representing group consensus, or the opinion of one contributor?

Reply:
@dhh1128, just my opinion, but I suspect others may take issue with the language. Saying a holder "never needs to reveal the verifier to the issuer" seems to unnecessarily cut out some use cases where the data model would work perfectly fine.

Something like: "Holders are able to share verifiable presentations without ever revealing the identity of the verifier to issuers" seems like better language because it talks about what we're enabling rather than trying to restrict use cases for the data model. In general, I think that's a better way to write spec language because specs can't ever stop people from behaving in certain ways. Rather, specs merely enable (or specify) new ways to do things interoperably.

Reply:
I am unable to see the difference between "never need to reveal the identity" vs "able to share without ever revealing the identity". Both mean that there is no circumstance where a holder **is required** to reveal the identity. Both statements don't imply that the holder **should not** or **can not**

Reply:
Changed the language